libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -358,6 +358,46 @@ export const expoConfig: IConfigCredentials[] = [
   ...pushConfigBase,
 ];
 
+export const pushwooshConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.ApplicationCode,
+    displayName: 'Application Code',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.NotificationTitle,
+    displayName: 'Notification Title',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Subscribe,
+    displayName: 'Auto Subscribe',
+    type: 'boolean',
+    required: true,
+  },
```

Comment:
Seems neither required based on the interface.
https://github.com/Pushwoosh/web-push-notifications/blob/92fcfb43bc6cab273a976f7b41ce5b7a21d01664/types/sdk/pushwoosh.d.ts#L135